demos/notebooks/demo_motion_correction.ipynb

```diff
@@ -393,9 +394,9 @@
    "outputs": [],
    "source": [
     "# print crispness values\n",
-    "print('Crispness original: '+ str(int(crispness_orig)))\n",
-    "print('Crispness rigid: '+ str(int(crispness_rig)))\n",
-    "print('Crispness elastic: '+ str(int(crispness_els)))"
+    "print('Crispness original: ' + str(int(crispness_orig)))\n",
```

Comment:
f-strings might be more elegant

Reply:
Yes this is a pretty old notebook it's third on my list to renovate eventually, but for now I think let's just leave it.